### PR TITLE
DUOS-1132[risk=no]Update question 1.3 on Dar Application page

### PR DESCRIPTION
**Pull request by raejohanek**
SCOPE:
When the profile is not filed out at all:
This box is disabled while simultaneously displaying the error that the DAR application cannot be submitted because it is a required field. Update the error message to direct the user to their profile. 

When the profile is filled out and the user is not a PI and does not have a PI: 
the box is pre-populated with ‘--’ allowing it to pass validation checks and the DAR is able to be submitted when it shouldn’t be. Remove the placeholder so the question errors and does not allow submission.

Addresses:
https://broadworkbench.atlassian.net/browse/DUOS-1132

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] PR is labeled with a Jira ticket number and includes a link to the ticket
- [ ] PR is labeled with a security risk modifier [no, low, medium, high] 
- [ ] PR describes scope of changes

In all cases:

- [ ] Get a minimum of one thumbs worth of review, preferably 2 if enough team members are available
- [ ] Get PO sign-off for all non-trivial UI or workflow changes
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/dar_application/ResearcherInfo.js [hunk 1]
```diff
@@ -228,7 +228,7 @@ export default function ResearcherInfo(props) {
               }),
               span({
                 className: 'cancel-color required-field-error-span', isRendered: (invalidInvestigator) && (showValidationMessages)
-              }, ['Required field'])
+              }, ['Required field, your Profile must have a PI'])
```
Comment (rushtong) [inline src/pages/dar_application/ResearcherInfo.js]: This, I think, is not technically true. A researcher can also BE the PI and in that case, not need to have a separate PI listed. In testing, it looks like this validation isn't working. I can turn off my PI status and have not PI listed:

----
Screen Shot 2021-03-26 at 9 40 06 AM

----
HOWEVER, I can still submit a DAR this way, so I don't think this message shows up. The PI field isn't selectable based on the user profile:

----
Screen Shot 2021-03-26 at 9 43 29 AM

----
I think this should be invalid from the start if the user's profile isn't the PI and doesn't have a PI. We need some form of messaging on the DAR form that explains this when the PI field isn't selectable like it is in the screenshot.

**Review by rushtong — APPROVED** [review]
This is fine as it is, but I think there is a bigger validation problem here. Feel free to merge this and we can tackle the validation issue in a new ticket.

File: src/pages/dar_application/ResearcherInfo.js (hunk 1 above)
Comment (raejohanek) [reply src/pages/dar_application/ResearcherInfo.js]: Good catch! I updated the comment to be more clear and removed the '--' placeholder that allows the DAR to be submitted when it shouldn't. I also updated the ticket to reflect this

File: src/pages/dar_application/ResearcherInfo.js [hunk 2]
```diff
@@ -150,7 +150,7 @@ export default function ResearcherInfo(props) {
               div({ isRendered: checkCollaborator !== true, className: 'display-inline' }, ['*']),
               div({ isRendered: checkCollaborator === true, className: 'display-inline italic' }, [' (optional)']),
               span({ className: 'default-color' },
-                ['Please autenticate with ',
+                ['Please authenticate with ',
```
Comment (rushtong) [inline src/pages/dar_application/ResearcherInfo.js:153]: Nice catch!

File: src/pages/dar_application/ResearcherInfo.js [hunk 3]
```diff
@@ -228,7 +228,7 @@ export default function ResearcherInfo(props) {
               }),
               span({
                 className: 'cancel-color required-field-error-span', isRendered: (invalidInvestigator) && (showValidationMessages)
-              }, ['Required field'])
+              }, ['Required field, ensure your Profile has a PI listed or marks you as the PI'])
```
Comment (rushtong) [inline src/pages/dar_application/ResearcherInfo.js]: Minor language nit, the researcher would have to mark themselves as a PI. This reads as if a PI must do something for the researcher. 

```suggestion
              }, ['Required field, ensure your Profile has a PI listed, or that you are selected as the PI'])
```

**Review by rushtong — APPROVED** [review]
Love the cleanup, thank you! One minor language suggestion inline. Feel free to merge if you want to make that change.

**Pull request merged**